src/utils/debounce.cljs

```diff
@@ -14,7 +14,7 @@
     (js/clearTimeout v)))
 
 (defn debounce-and-dispatch
-  "Dispatches `event` iff it was not dispatched for the duration of `duration-ms`."
+  "Dispatches `event` if it was not dispatched for the duration of `duration-ms`."
```

Comment:
it was technically correct before 🤓 https://en.wikipedia.org/wiki/If_and_only_if

Reply:
My bad 🤦 Will revert it 👍

Reply:
Oh it was just a nerdy joke. Definitely a typo :)